submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/model/SubmissionStatType.kt

```diff
@@ -1,5 +1,5 @@
 package ac.uk.ebi.biostd.persistence.common.model
 
 enum class SubmissionStatType {
-    VIEWS
+    VIEWS, FILES_SIZE
```

Comment:
I am wondering if there is value in this? is not easier to leave stats types to api user/consumer. Backend does not know anything about what is stored there but rather only do it.

Reply:
I don't think it's a good idea to just leave any value open to be registered, especially since we already know which kind of statistics we want to keep a record of.

Allowing any kind of data would open the door to a lot of data inconsistencies since the stats are something we want to know about all submissions, not generic data that could be different on submission basis